<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The Current Sgtk Environment</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="context_overview_tab_widget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icon_task.png</normaloff>:/res/icon_task.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Current Context</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="ContextBrowserWidget" name="context_browser" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>0</height>
          </size>
         </property>
         <property name="accessibleName">
          <string>context_browser</string>
         </property>
         <property name="accessibleDescription">
          <string>context_browser</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Current Context lists all ShotGrid objects that together make up the Current Work Area. Double-click an item to access the object in ShotGrid.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/logo_color_16.png</normaloff>:/res/logo_color_16.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Active Apps</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="AppBrowserWidget" name="app_browser" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>0</height>
          </size>
         </property>
         <property name="accessibleName">
          <string>app_browser</string>
         </property>
         <property name="accessibleDescription">
          <string>app_browser</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Active Apps lists all currently active apps. Double-click an app to jump to its documentation.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/cog_white.png</normaloff>:/res/cog_white.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Environment</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="EnvironmentBrowserWidget" name="environment_browser" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>0</height>
          </size>
         </property>
         <property name="accessibleName">
          <string>environment_browser</string>
         </property>
         <property name="accessibleDescription">
          <string>environment_browser</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>The Environment file contains all settings and configurations for currently running ShotGrid apps. The Engine provides core services, such as menu management and app startup.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/res/tank_logo.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="jump_to_fs">
       <property name="text">
        <string>Jump to the File System</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reload_apps">
       <property name="text">
        <string>Reload Engine and Apps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="support">
       <property name="text">
        <string>Documentation and Support</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContextBrowserWidget</class>
   <extends>QWidget</extends>
   <header>..context_browser</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AppBrowserWidget</class>
   <extends>QWidget</extends>
   <header>..app_browser</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EnvironmentBrowserWidget</class>
   <extends>QWidget</extends>
   <header>..environment_browser</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
